project/make/gccgo

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+set -e
+
+DEST=$1
+BINARY_NAME="docker-$VERSION"
+BINARY_EXTENSION="$(binary_extension)"
+BINARY_FULLNAME="$BINARY_NAME$BINARY_EXTENSION"
+
+source "$(dirname "$BASH_SOURCE")/.go-autogen"
+
+if [ "$IAMSTATIC" == "true" ]; then 
+	EXTLDFLAGS_STATIC_DOCKER="-static -Wl,--no-export-dynamic $(echo $EXTLDFLAGS_STATIC_DOCKER | sed "s/$EXTLDFLAGS_STATIC//")"
+else
+	EXTLDFLAGS_STATIC_DOCKER="-static-libgo $(echo $EXTLDFLAGS_STATIC_DOCKER | sed "s/$EXTLDFLAGS_STATIC//")"
```

Comment:
I don't think we really need to statically link against libgo by using -static-libgo.  Setting EXTLDFLAGS_STATIC_DOCKER to null in dynbinary_gccgo seems to be the right approach to me